geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -1372,14 +1368,13 @@ private void initializeDeclarativeCache()
     } catch (CacheXmlException ex) {
       CacheXmlException newEx =
           new CacheXmlException(String.format("While reading Cache XML %s. %s",
-              url, ex.getMessage()));
+              url, ex.getMessage()), ex.getCause());
       /*
        * TODO: why use setStackTrace and initCause? removal breaks several tests: OplogRVVJUnitTest,
```

Comment:
This comment needs to be updated

Reply:
Oh, good catch. I had corrected it once but I did some undoes and missed this again.

Reply:
Turns out that the whole comment is false. All those tests pass without overriding the default stack track too. I will clean it all up.